HW_10/task_2_fibonacci_number.py

```diff
@@ -0,0 +1,37 @@
+# Task 2. (optional) Create a program that will accept a number and
+# print the corresponding number in the Fibonacci sequence using recursion.
+
+def print_fibonacci_sequence(number, lst):
+    print(f'The number in Fibonacci Sequence: {number}')
+    print(f'The Fibonacci Sequence to the number: {lst}')
+
+
+def get_fibonacci_number(number, lst=None):
```

Comment:
nit: seems too complex, could be implemented like
```suggestion
def fibonacci(n):
    if n <= 1:
        return n
    return fibonacci(n - 1) + fibonacci(n - 2)
```